<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" ignored="0" total="8" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="6856" started-at="2017-08-16T10:36:34Z" finished-at="2017-08-16T10:36:40Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="6856" started-at="2017-08-16T10:36:34Z" finished-at="2017-08-16T10:36:40Z">
      <class name="TestCases.Test_RealmAPI">
        <test-method status="PASS" signature="CreateRealmAPIPost()[pri:1, instance:TestCases.Test_RealmAPI@527740a2]" name="CreateRealmAPIPost" duration-ms="2463" started-at="2017-08-16T16:06:34Z" finished-at="2017-08-16T16:06:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateRealmAPIPost -->
        <test-method status="PASS" signature="DuplicateCreateRealmAPI()[pri:2, instance:TestCases.Test_RealmAPI@527740a2]" name="DuplicateCreateRealmAPI" duration-ms="455" started-at="2017-08-16T16:06:36Z" finished-at="2017-08-16T16:06:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DuplicateCreateRealmAPI -->
        <test-method status="PASS" signature="CreateRealmAPIPostWithEmptyName()[pri:3, instance:TestCases.Test_RealmAPI@527740a2]" name="CreateRealmAPIPostWithEmptyName" duration-ms="405" started-at="2017-08-16T16:06:37Z" finished-at="2017-08-16T16:06:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateRealmAPIPostWithEmptyName -->
        <test-method status="PASS" signature="CreateRealmAPIPostWithMaxLengthName()[pri:4, instance:TestCases.Test_RealmAPI@527740a2]" name="CreateRealmAPIPostWithMaxLengthName" duration-ms="429" started-at="2017-08-16T16:06:37Z" finished-at="2017-08-16T16:06:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateRealmAPIPostWithMaxLengthName -->
        <test-method status="FAIL" signature="CreateRealmAPIPostWithMaxLengthDescription()[pri:5, instance:TestCases.Test_RealmAPI@527740a2]" name="CreateRealmAPIPostWithMaxLengthDescription" duration-ms="411" started-at="2017-08-16T16:06:37Z" finished-at="2017-08-16T16:06:38Z">
          <exception class="java.lang.StringIndexOutOfBoundsException">
            <message>
              <![CDATA[String index out of range: -6]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.StringIndexOutOfBoundsException: String index out of range: -6
	at java.lang.String.substring(Unknown Source)
	at TestCases.Test_RealmAPI.CreateRealmAPIPostWithMaxLengthDescription(Test_RealmAPI.java:170)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.StringIndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateRealmAPIPostWithMaxLengthDescription -->
        <test-method status="FAIL" signature="GetRealmAPI()[pri:6, instance:TestCases.Test_RealmAPI@527740a2]" name="GetRealmAPI" duration-ms="1016" started-at="2017-08-16T16:06:38Z" finished-at="2017-08-16T16:06:39Z">
          <exception class="org.jsoup.HttpStatusException">
            <message>
              <![CDATA[HTTP error fetching URL]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.jsoup.HttpStatusException: HTTP error fetching URL. Status=404, URL=http://recruit01.test01.brighttalk.net:8080/user/realm1144
	at org.jsoup.helper.HttpConnection$Response.execute(HttpConnection.java:537)
	at org.jsoup.helper.HttpConnection$Response.execute(HttpConnection.java:493)
	at org.jsoup.helper.HttpConnection.execute(HttpConnection.java:205)
	at org.jsoup.helper.HttpConnection.get(HttpConnection.java:194)
	at TestCases.Test_RealmAPI.GetRealmAPI(Test_RealmAPI.java:198)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- org.jsoup.HttpStatusException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GetRealmAPI -->
        <test-method status="FAIL" signature="GetRealmAPIForIdMaxLengthVerification()[pri:7, instance:TestCases.Test_RealmAPI@527740a2]" name="GetRealmAPIForIdMaxLengthVerification" duration-ms="579" started-at="2017-08-16T16:06:39Z" finished-at="2017-08-16T16:06:40Z">
          <exception class="org.jsoup.HttpStatusException">
            <message>
              <![CDATA[HTTP error fetching URL]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.jsoup.HttpStatusException: HTTP error fetching URL. Status=500, URL=http://recruit01.test01.brighttalk.net:8080/user/realm/114452
	at org.jsoup.helper.HttpConnection$Response.execute(HttpConnection.java:537)
	at org.jsoup.helper.HttpConnection$Response.execute(HttpConnection.java:493)
	at org.jsoup.helper.HttpConnection.execute(HttpConnection.java:205)
	at org.jsoup.helper.HttpConnection.get(HttpConnection.java:194)
	at TestCases.Test_RealmAPI.GetRealmAPIForIdMaxLengthVerification(Test_RealmAPI.java:216)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- org.jsoup.HttpStatusException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GetRealmAPIForIdMaxLengthVerification -->
        <test-method status="FAIL" signature="GetRealmAPIForNonExistingRealmID()[pri:8, instance:TestCases.Test_RealmAPI@527740a2]" name="GetRealmAPIForNonExistingRealmID" duration-ms="861" started-at="2017-08-16T16:06:40Z" finished-at="2017-08-16T16:06:40Z">
          <exception class="org.jsoup.HttpStatusException">
            <message>
              <![CDATA[HTTP error fetching URL]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.jsoup.HttpStatusException: HTTP error fetching URL. Status=400, URL=http://recruit01.test01.brighttalk.net:8080/user/realm/5000
	at org.jsoup.helper.HttpConnection$Response.execute(HttpConnection.java:537)
	at org.jsoup.helper.HttpConnection$Response.execute(HttpConnection.java:493)
	at org.jsoup.helper.HttpConnection.execute(HttpConnection.java:205)
	at org.jsoup.helper.HttpConnection.get(HttpConnection.java:194)
	at TestCases.Test_RealmAPI.GetRealmAPIForNonExistingRealmID(Test_RealmAPI.java:235)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- org.jsoup.HttpStatusException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GetRealmAPIForNonExistingRealmID -->
      </class> <!-- TestCases.Test_RealmAPI -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
